src/Registration.js

```diff
@@ -0,0 +1,22 @@
+import FormInput from "./FormInput";
```

Comment:
Please, put that file to the folder `/src/components/PasswordWithConfirm` and make it .tsx